okonomiyaki/utils/tests/test_test_data.py

```diff
@@ -0,0 +1,47 @@
+import glob
+from os.path import join, dirname
+import sys
+import zipfile2
+
+from okonomiyaki.runtimes.runtime_metadata import IRuntimeMetadata
+from okonomiyaki.errors import UnsupportedMetadata
+
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
+
+
+class TestDummyPythonRuntimes(unittest.TestCase):
+
+    def _get_contents_of_runtime(self, runtime):
+        with zipfile2.ZipFile(runtime) as zp:
+            return zp.namelist()
+
+    def test_pythonw_in_dummy_runtime(self):
+        """
+        Ensure that pythonw.exe is included in all Windows runtimes
+        """
+
+        # Given
+        test_data_dir = join(dirname(__file__), '..', 'test_data')
```

Comment:
Can we instead have a `DUMMY_RUNTIMES_DIRECTORY` in okonomiyaki.utils.test_data that points to this instead ?